tests/images/customize.sh

```diff
@@ -145,45 +145,106 @@ qemu_disk() {
 }
 
 assert() {
```

Comment:
One small suggestion, how about adding fun `assert_not(){}`, then it would be not confused to get logs about `Assertion failed:` when call `! assert xx`.

Reply:
good idea! updated

Reply:
Then this can be removed.